### PR TITLE
update faq.md, add new question

### PR DESCRIPTION
**Pull request by idc77**
question: How to change the character set and/or collation of a MySQL table?

Not sure if the slug path is correct.

### CODE REVIEW

File: doc/md/faq.md [hunk 1]
```diff
@@ -759,3 +760,22 @@ func (d *multiDriver) Close() error {
 }
 ```
 
+#### How to change the character set and/or collation of a MySQL table?
+
+By default for MySQL the character set `utf8mb4` is used and the collation of `utf8mb4_bin`.
+However if you'd like to change the schema's character set and/or collaction you will need to use the annotation feature.
```
Comment (masseelch) [inline doc/md/faq.md:766]:
```suggestion
However if you'd like to change the schema's character set and/or collation you need to use an annotation.
```

**Review by masseelch — APPROVED** [review]
Thanks!

File: doc/md/faq.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ sidebar_label: FAQ
 [How to add `CHECK` constraints to table?](#how-to-add-check-constraints-to-table)  
 [How to define a custom precision numeric field?](#how-to-define-a-custom-precision-numeric-field)  
 [How to configure two or more `DB` to separate read and write?](#how-to-configure-two-or-more-db-to-separate-read-and-write)
```
Comment (a8m) [inline doc/md/faq.md:24]:
```suggestion
[How to configure two or more `DB` to separate read and write?](#how-to-configure-two-or-more-db-to-separate-read-and-write)  
```

Comment (a8m) [reply doc/md/faq.md:24]: A line break is needed here.

**Comment by a8m** [comment]
Thanks for your contribution @idc77 🙏 ❤️ 

@all-contributors please add @idc77 for docs.

**Pull request merged**